### PR TITLE
Amend CLI globbing help text (7.8.x)

### PR DESCRIPTION
**Pull request by hjoliver**
7.8.x companion of #3302

-----
This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (CLI help text only).
- [x] Appropriate change log entry included.
- [x] No documentation update required (is documentation).

### CODE REVIEW

File: lib/cylc/option_parsers.py [hunk 1]
```diff
@@ -32,12 +32,16 @@ class CylcOptionParser(OptionParser):
...
     MULTITASK_USAGE = """
-TASKID is a pattern to match task proxies or task families, or groups of them:
+TASKID is a pattern to match task proxies, or families, or groups of them:
 * [CYCLE-POINT-GLOB/]TASK-NAME-GLOB[:TASK-STATE]
 * [CYCLE-POINT-GLOB/]FAMILY-NAME-GLOB[:TASK-STATE]
 * TASK-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
 * FAMILY-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
 
+WARNING: this command matches and operates on task proxy instances in the
+scheduler task pool, NOT abstract tasks in the workflow. If a task is not
```
Comment (sadielbartholomew) [inline lib/cylc/option_parsers.py]: Maybe this instead (for this branch), to remain consistent with Cylc 7 terminology where we have not generalised to 'workflow'?

```suggestion
scheduler task pool, NOT abstract tasks in the suite. If a task is not
```

File: CHANGES.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ Selected user-facing changes:
 
 ### Enhancements
 
+[#3303](https://github.com/cylc/cylc-flow/pull/3259) - improve CLI
```
Comment (sadielbartholomew) [inline CHANGES.md]: It seems you missed out the change on the PR ID from a copy 'n paste:

```suggestion
[#3303](https://github.com/cylc/cylc-flow/pull/3303) - improve CLI
```

File: lib/cylc/option_parsers.py (hunk 1 above)
Comment (hjoliver) [reply lib/cylc/option_parsers.py]: Fair enough!

File: CHANGES.md (hunk 2 above)
Comment (hjoliver) [reply CHANGES.md]: Oops, good spotting!

**Comment by hjoliver** [comment]
@sadielbartholomew - I agree with changing the order as you suggest; will amend the branch tomorrow.

**Comment by hjoliver** [comment]
Hopefully good to go now @sadielbartholomew. I ended up trying to fix some additional ambiguities in task-targetting CLI help docs.

**Comment by hjoliver** [comment]
One genuine test failure, damn it.

**Comment by hjoliver** [comment]
(But the PR has not updated with my last pushed commit ... GitHub reacting slowly :thinking: )

**Comment by sadielbartholomew** [comment]
> GitHub reacting slowly

There's a major outage according to GH status :( I'll wait upon that final commit before reviewing prematurely again!

**Comment by hjoliver** [comment]
GitHub booted to updated the PR; tests all passed :tada:

**Pull request merged**